### PR TITLE
Postgres dump

### PR DESCRIPTION
**Pull request by se3000**
Rake commands to easily create a pg_dump and restore from it. Based on [Heroku's format](https://devcenter.heroku.com/articles/heroku-postgres-import-export) for pg:backups.

### CODE REVIEW

**Comment by se3000** [comment]
Also working on a command to automate torrent creation, but that part's a bit trickier to test; will send in soon.

Figured this might be helpful in the meantime.

File: Rakefile [hunk 1]
```diff
@@ -19,19 +35,27 @@ namespace :db do
...
+  desc "Create test and dev databases"
+  task create: [:set_pg_password] do
     sh "dropdb #{sys_db_args} #{db_name} || true" # drop if exists
     sh "createdb #{sys_db_args} #{db_name}"
     Rake::Task['db:migrate'].invoke
   end
+
+  desc "Create a copy of the database"
+  task dump: [:set_pg_password] do
+    sh "pg_dump -Fc --no-acl --no-owner #{sys_db_args} #{db_name} > latest.dump"
```
Comment (aalness) [inline Rakefile]: This is great. My only comment is the path should be an argument.

**Comment by aalness** [comment]
Awesome. Will merge as soon as tests pass. I seem to have introduced a flakey test with my BIP66 changes.

**Comment by aalness** [comment]
Actually, could you please squash into one commit?

**Pull request merged**